paddle/fluid/operators/jit/test.cc

```diff
@@ -761,7 +821,15 @@ TEST(JITKernel, kNCHW16CMulNC) {
   TestNCHW16CMulNCKernel<jit::kNCHW16CMulNC, double, CPUPlace>();
 }
 
-// TODO(yihua/TJ): add crf decoding and layer norm unit tests
+TEST(JITKernel, kLayerNorm) {
+  namespace jit = paddle::operators::jit;
+  TestLayerNormKernel<jit::kLayerNorm, float, paddle::platform::CPUPlace>();
```

Comment:
Test should contain both `float` and `double`, Benchmark should only focus on `float`.

Reply:
If we add the double's support, we need implement the correspond kernel. Can we implement it later?

Reply:
Use the reference path to replace double type's support